test/utils/wabt_engine.cpp

```diff
@@ -8,17 +8,29 @@
 
 #include <test/utils/adler32.hpp>
 #include <test/utils/wasm_engine.hpp>
+#include <algorithm>
 #include <cassert>
 
 namespace fizzy::test
 {
-class WabtEngine final : public WasmEngine
+namespace
+{
+const wabt::Features& get_features() noexcept
```

Comment:
This is cheap object so can be returned by value. I guess it is taken by reference by other functions?